data-source/jsonnet/northern-ireland/blocks/individual/identity-and-health/religion.jsonnet

```diff
@@ -73,8 +81,8 @@ local question(title) = {
         block: 'no-religion',
         when: [
           {
-            id: 'religion-answer',
-            condition: 'equals',
+            id: 'religion-answer-exclusive',
+            condition: 'contains',
```

Comment:
Up to you, if you use `condition: 'set',` and remove the `value` line then it would work the same way and it's 1 less place to change content that could be missed.

Reply:
Given how we're using this mutually exclusive (the only option in that second group is the no/none option), that would work, yes, good idea.

Reply:
done